composer.json

```diff
@@ -4,10 +4,11 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
+        "drush/drush": "^10.3",
         "drupal/purge": "^3.0@beta",
         "drupal/purge_queuer_url": "^1.0@RC",
         "drupal/redis": "^1.0@RC",
-        "drupal/section_purger": "0.2.3",
+        "drupal/section_purge": "2.3-rc2",
```

Comment:
Has the name of the module actually changed?

Reply:
Just confirmed - yes it has.

- OK: https://www.drupal.org/project/section_purge
- 404: https://www.drupal.org/project/section_purger